src/Training.AirBnb.Clone.Backend/AirBnB.Application/Common/Settings/BackgroundServiceSettings.cs

```diff
@@ -0,0 +1,12 @@
+namespace AirBnB.Application.Common.Settings;
+
+/// <summary>
+/// Represents the settings for a background service.
+/// </summary>
+public class BackgroundServiceSettings
+{
+    /// <summary>
+    /// Gets or sets the time interval between each execution of the Listing Rating recalculation process, specified in hours.
+    /// </summary>
+    public int ListingRatingRecalculationIntervalInHours { get; set; }
+}
```

Comment:
Need to use this as general configuration model for background services, and change property name to `ExecutionIntervalInSeconds` to be able to set interval precisely

And create another configuration model specifically for rating recalculation